packages/edit-site/src/components/block-editor/style.scss

```diff
@@ -27,14 +27,6 @@
 	// Centralize the editor horizontally (flex-direction is column).
 	align-items: center;
 
-	// Controls height of editor and editor canvas container (style book, global styles revisions previews etc.)
-	iframe {
```

Comment:
The styles are the same across editors so I centralized them in packages/block-editor/src/components/block-canvas/style.scss.

If there's any reason to have bespoke styles I guess they can be overwritten in the site or post editor packages (?)